apps/ai_reports/signals.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+import httpx
+from django.conf import settings
+from django.db.models import signals
+from django.dispatch import receiver
+
+from adhocracy4.comments.models import Comment
+from apps.ai_reports import tasks
+
+client = httpx.Client()
+logger = logging.getLogger(__name__)
+
+
+@receiver(signals.post_save, sender=Comment)
+def get_ai_classification(sender, instance, created, update_fields, **kwargs):
+    if getattr(settings, "XAI_ENABLE", False):
+        if getattr(settings, "XAI_API_URL", None):
```

Comment:
If I use `hasattr(settings, "XAI_ENABLE")` it will also be true if you set `XAI_ENABLE=False` in your settings, that doesn't seem like a good idea to me ?

Reply:
You only added it in testing.py, so I assumed we won't add any setting in base.txt. If we add XAI url as an empty string, and yes one setting is fine, then the gettatr makes sense.